src/components/drought-selector.tsx

```diff
@@ -1,28 +1,29 @@
 import React from "react";
 import { Slider } from "@material-ui/core";
 import VerticalHandle from "../assets/slider-vertical.svg";
-import NoDrought from "../assets/terrain/drought-no.svg";
-import MildDrought from "../assets/terrain/drought-mild.svg";
-import MedDrought from "../assets/terrain/drought-med.svg";
-import SevereDrought from "../assets/terrain/drought-severe.svg";
 
 import { generateMarks, droughtIcons, droughtLabels } from "./vertical-selectors";
 import * as css from "./vertical-selectors.scss";
+import { DroughtLevel } from "../models/fire-model";
```

Comment:
In general, we need `DroughtLevel` as the same levels will result in different moisture content for different vegetation types, right?